src/core/mesh/qgsmeshlayer.h

```diff
@@ -728,6 +730,26 @@ class CORE_EXPORT QgsMeshLayer : public QgsMapLayer
     */
     bool startFrameEditing( const QgsCoordinateTransform &transform );
 
+    /**
+    * Commit edition of the mesh frame,
+    * Rebuilds the triangular mesh and its spatial index with \a transform,
+    * Continue editing with the same mesh editor if \a continueEditing is True
```

Comment:
```suggestion
    * Continue editing with the same mesh editor if \a continueEditing is TRUE
```

(if you use capitals for TRUE/FALSE/NULLPTR in dox then they are automatically converted to true/false/nullptr for c++ dox and True/False/None for Python docs)